### PR TITLE
add unittest for Exec

### PR DESCRIPTION
**Pull request by yxxhero**
add unittest for Exec

Signed-off-by: yxxhero <aiopsclub@163.com>

### CODE REVIEW

**Review by itscaro — APPROVED** [review]

File: pkg/tmpl/context_funcs.go [hunk 1]
```diff
@@ -66,6 +66,8 @@ func (c *Context) Exec(command string, args []interface{}, inputs ...string) (st
 
 	cmd := exec.Command(command, strArgs...)
 	cmd.Dir = c.basePath
+	// inherit the environment variables from the parent process
+	cmd.Env = os.Environ()
```
Comment (mumoshu) [inline pkg/tmpl/context_funcs.go:70]: @yxxhero Have you tried it without this change? I thought Go's `exec.Command` implicitly inherited the envvars from the process when `cmd.Env == nil`.

Comment (yxxhero) [reply pkg/tmpl/context_funcs.go:70]: I will test it without this change. wait a monment.

Comment (yxxhero) [reply pkg/tmpl/context_funcs.go:70]: @mumoshu you are right. Thanks very much.
```yaml
If Env is nil, the new process uses the current process's environment.
```

**Review by mumoshu — APPROVED** [review]
LGTM. Thanks a lot for your continued effort ☺️

**Pull request merged**